dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/scheduling/JobScheduler.java

```diff
@@ -242,6 +242,7 @@ private void runDueJob(JobConfiguration config, Instant start) {
         service.finishRunCancel(jobId);
       }
     } catch (Exception ex) {
+      if (progress != null) progress.failedProcess(ex);
```

Comment:
FYI: This is an improvement so that the progress tracking also will have status ERROR when the job is considered as FAILED/ERROR (`lastExecutionStatus`)